transcendence-app/src/user/dto/user.dto.ts

```diff
@@ -30,3 +31,15 @@ export class UserDto {
   @ValidateIf((object, value) => value !== null)
   avatarId!: string | null;
 }
+
+export class UserDto {
+  username!: string;
+  email!: string;
+  fullName!: string;
+  avatarId!: string | null;
+  avatarX!: number;
+  avatarY!: number;
+  id!: string;
+  createdAt!: Date;
+  isBlocked?: boolean | null;
+}
```

Comment:
We response with a DTO, today we've added the blocked info but tomorrow we could attach other valuable information to this or other DTOs